pkg/netlink/monitor.go

```diff
@@ -0,0 +1,159 @@
+package netlink
```

Comment:
why did you add that to their package?
should probably be internal - just in case they will add it late.
unrelated, you may submit a PR to the netlink repo for adding this?